ckanext/canada/templates/scheming/package/snippets/resource_form.html

```diff
@@ -11,41 +11,31 @@
-  {% endif %}
   <div class="accordion mrgn-tp-lg mrgn-bttm-lg">
-    {%- if (not data.related_type) or g.action.endswith('new') -%}
-      <details class="acc-group" open="open">
-        <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group"}'>
-          {{ _("Metadata Fields for Resources Only (Not for Related Items)") }}
-        </summary>
-        <div class="tgl-panel">
-          {%- for field in schema.resource_fields -%}
-            {%- if field.form_panel == 'resource' -%}
-              {%- snippet 'scheming/snippets/form_field.html', field=field, data=data, errors=errors, entity_type='dataset', object_type=dataset_type -%}
-            {%- endif -%}
-          {%- endfor -%}
-        </div>
-      </details>
-    {% endif %}
-    {%- if data.related_type or g.action.endswith('new') -%}
-      <details class="acc-group" {% if data.related_type %}open="open"{% endif %}>
-        <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group"}'>
-          {{ _("Metadata Fields for Related Items Only") }}
-        </summary>
-        <div class="tgl-panel">
-          {%- for field in schema.resource_fields -%}
-            {%- if field.form_panel == 'related' -%}
-              {%- snippet 'scheming/snippets/form_field.html', field=field, data=data, errors=errors, entity_type='dataset', object_type=dataset_type -%}
-            {%- endif -%}
-          {%- endfor -%}
-          <p>
-            {{ _("Use the Download URL field above to link to the related record.") }}
-          </p>
-        </div>
-      </details>
-    {% endif %}
+    <details id="normal-resource-meta-group" class="acc-group-res" open="open">
+      <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group-res"}'>
+        {{ _("Metadata Fields for Resources") }}
+      </summary>
+      <div class="tgl-panel">
+        {%- for field in schema.resource_fields -%}
+          {%- if field.form_panel == 'resource' -%}
+            {%- snippet 'scheming/snippets/form_field.html', field=field, data=data, errors=errors, entity_type='dataset', object_type=dataset_type -%}
+          {%- endif -%}
+        {%- endfor -%}
+      </div>
+    </details>
+    <details id="related-resource-meta-group" class="acc-group-rel" open="open">
+      <summary class="wb-toggle tgl-tab" data-toggle='{"parent": ".accordion", "group": ".acc-group-rel"}'>
+        {{ _("Metadata Fields for Resource Relationships") }}
+      </summary>
+      <div class="tgl-panel">
+        {%- for field in schema.resource_fields -%}
+          {%- if field.form_panel == 'relationships' -%}
+            {%- snippet 'scheming/snippets/form_field.html', field=field, data=data, errors=errors, entity_type='dataset', object_type=dataset_type -%}
+          {%- endif -%}
+        {%- endfor -%}
+      </div>
+    </details>
```

Comment:
Before resources were used *as* relationships for datasets because we didn't have repeating subfields.

Now resource relationships are relationships between resources, and the dataset relationships are defined in the dataset form. Will it be confusing for users to have relationships defined at both levels? Are there many cases where users will want to define relationships between resources (and if they do, maybe there are too many resources in a dataset?)

Reply:
Well the original request for bringing this all back was that Pat wanted to have references between Resources. E.g. this resource is a JSON schema which DEFINES this other resource which is a dataset.

That is not what the old way was, it was a resource being used to have some relationship to the dataset that the resource belonged to. So did not meet the requirement Pat wanted. So the original resource as relationship is moved to the repeating fields in the Dataset. And the new requirement that Pat wants are the repeating fields in the Resources.

I think both make sense. Are they confusing? absolutely, but so was the old resource as a relationship, it didn't even really make much sense and using a resource as not really a resource was bad. These at least make more sense. There will for sure need to be updates to the Registry Operations Guide for this.

But the Dataset repeating fields make a lot of sense for all of those users that keep segmenting their datasets for fiscal years. E.g. Fish Data 2024 continues Fish Data 2023, and/or Fish Data 2023 is Continued by Fish Data 2024